packages/jsii-pacmak/lib/targets/version-utils.ts

```diff
@@ -65,6 +65,10 @@ function toBracketNotation(
     .map((set) => {
       if (set.length === 1) {
         const version = set[0].semver.raw;
+        if (!version) {
+          // Case where version is '*'
+          return `(0.0.0,]`;
```

Comment:
This probably should be *inclusive* of `0.0.0`.

```suggestion
          return `[0.0.0,]`;
```

I kinda wonder if the '*' shouldn't be detected upfront, and maybe "replaced" with `>= 0.0.0` instead?

Reply:
I think it's better to detect it here within JSII (rather than in `packages/jsii-pacmak/lib/targets/java.ts`, etc), so it's just done once, rather than in both the maven-related code and nuget-related code.

I also don't think consumers should have to worry about this, I think style-wise `*` is more common than `>=0.0.0`.

Reply:
Oh no you misunderstood me! I meant catch it at the beginning of *this function*, adding something along the lines of:

```
if (semverRange === '*') {
  semverRange = '>=0.0.0';
}
```

And then letting the rest of the code work it's magic unchanged.

Reply:
Ahh! Will do that :)

Reply:
The problem I'm running into here is that:

```
new Range('>=0.0.0');
```

leads to this:

```
     Range {
        options: { loose: false, includePrerelease: false },
        loose: false,
        includePrerelease: false,
        raw: '>=0.0.0',
        set: [ [ [Comparator] ] ],
        range: '' }
```

And `range.set[0].semver` is `Symbol(SemVer ANY)`

and thus `range.set[0].semver.raw` is undefined :/

Reply:
Well this sucks! I guess you've tried 🤷🏻‍♂️

Reply:
Would you suggest a mix of checking in both places, or just where I had previously? I can also check `range.raw>=0.0.0` (instead of just `set[0].semver.raw` undefined